### DISCUSSION TITLE
Fake Paypal Link

### DISCUSSION THREAD

**Discussion by Tilyblu** [PrestaShop/PrestaShop#31816] [Q&A] [answered]
Hello, I have a fake PayPal link on my website. I have tried severally to unhook it. Can anyone please advise what to do?

**Comment by matks** [comment, marked as answer]
Hello if your website has been hacked and a fake paypal link has been inserted on it I would suggest to reach out to a professional. There are thousands of agencies and freelance experts located around the globe, see https://www.prestashop-project.org/support/
